guides/common/modules/proc_customizing-registration-by-using-snippets.adoc

```diff
@@ -0,0 +1,23 @@
+[id="customizing-registration-by-using-snippets"]
+= Customizing registration by using snippets
+
+You can customize the registration process without editing the *Global Registration* template by creating snippets that the template includes if they exist.
+To add custom steps to registration, create one or both of the following snippets:
+
+* `before_registration` {endash} This snippet will be loaded and executed by the *Global Registration* template at the beginning of registration.
+* `after_registration` {endash} This snippet will be loaded and executed by the *Global Registration* template at the end of registration.
+
+You have to name the snippets precisely.
```

Comment:
```suggestion
Make sure you name the snippets precisely.
```
Feels more direct :shrug:

Reply:
Also, how about turning this into an IMPORTANT admonition?

Reply:
Well, we used to avoid admonitions really hard in order not to "overadmonition" the docs. So I'm not sure what stand to take on this.
@maximiliankolb Opinion?

Reply:
I don't think an admonition is necessary here. Maybe "Ensure that you have named the snippet `X`."?

Reply:
I think it's clear enough.